client/core/core.go

```diff
@@ -3549,39 +3607,51 @@ func (c *Core) resumeTrades(dc *dexConnection, trackers []*trackedTrade) assetMa
 			}
 		}
 
-		// Active orders and orders with matches with unsent swaps need the funding
-		// coin(s).
+		// Active orders and orders with matches with unsent swaps need funding
+		// coin(s). If they are not found, block new matches and swap attempts.
+		needsCoins := len(matchesNeedingCoins) > 0
 		if isActive || needsCoins {
 			coinIDs := trade.Coins
 			if len(tracker.metaData.ChangeCoin) != 0 {
 				coinIDs = []order.CoinID{tracker.metaData.ChangeCoin}
 			}
+			tracker.coins = map[string]asset.Coin{} // should already be
```

Comment:
This really shouldn't be necessary here more than any other place we use a map. Did you see a problem?

Reply:
`swapMatchGroup` has both a map lookup and an insert, so the map needs to be initialized.  There's a comment in `newTrackedTrade` stating that it must not be nil as well.